server/dbSchemas/attendanceSchema.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+    email: { type: String, required:true},
+    name: { type: String, required: true },
+    password: { type: String, required: true },
+    is_student: { type: Boolean, required: true},
+    is_professor: { type: Boolean, required: true },
+    student_number: String
+})
+
+const courseSchema = new Schema({
+    name: { type: String, required: true },
+    admin: { type: userSchema._id, required: true },
+    date_created: { type: Date, required: true }
+})
+
+const sectionSchema = new Schema({
```

Comment:
Would also need to store the classroom layout for the section, will speak to the DB team to further clarify, but this can be added in a later pull request